### PR TITLE
Scheduler refactoring

### PR DESCRIPTION
**Pull request by flexik**
I have refactored scheduler script a bit to use `sirius:events` rake task instead of calling ScheduleManager directly.

Also the logging gem was added to improve logging in scheduler and rake tasks.

### CODE REVIEW

File: config/initializers/logging.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Logging configuration for the whole app.
+#
+# For details see https://github.com/TwP/logging
+
+require 'logging'
+
+Logging.logger.root.appenders = Logging.appenders.stdout(
+  layout: Logging::Layouts.pattern(format_as: :inspect, pattern: '%-5l [%c]: %m\n')
+)
+Logging.logger.root.level = (ENV['RACK_ENV'] == 'production' ? :info : :debug)
```
Comment (jnv) [inline config/initializers/logging.rb]: I _suppose_ this could use `Config.rack_env` so there's a guaranteed default, but this is also fine.

**Comment by jnv** [comment]
My original idea was that you would put the shared sequence into ScheduleManager, but I suppose this can also work. Just make sure invocation from scheduler does work.

File: config/initializers/logging.rb (hunk 1 above)
Comment (jirutka) [inline config/initializers/logging.rb]: Agree, I’ve changed it.

**Pull request merged**